framework/test/unit/modules/chain/unit/helpers/init_transaction.js

```diff
@@ -28,209 +28,198 @@ const initTransaction = require('../../../../../../src/modules/chain/helpers/ini
 
 describe('init transaction', async () => {
 	it('should initialize a transfer transaction', async () => {
```

Comment:
this `it` seems un-aligned with other test

Reply:
how so?

Reply:
spacing seems in correct? maybe only in github